packages/server/graphql/public/types/UpdatedNotification.ts

```diff
@@ -0,0 +1,20 @@
+import {getUserId} from '../../../utils/authorization'
+import {UpdatedNotificationResolvers} from '../resolverTypes'
+
+export type UpdatedNotificationSource = {updatedNotificationId: string}
+
+const UpdatedNotification: UpdatedNotificationResolvers = {
+  updatedNotification: async ({updatedNotificationId}, _args: unknown, {dataLoader, authToken}) => {
+    const viewerId = getUserId(authToken)
+    const notification = await dataLoader.get('notifications').load(updatedNotificationId)
+    if (notification.userId !== viewerId) {
+      throw new Error(
+        `Viewer ID does not match notification user ID: notification ${updatedNotificationId} for user ${notification.userId} published to user ${viewerId}`
```

Comment:
For future reference: Errors are not user facing because they're sanitized: https://github.com/ParabolInc/parabol/blob/master/packages/server/utils/sanitizeGraphQLErrors.ts